app/src/main/java/com/example/calculator/MainActivity.kt

```diff
@@ -2,10 +2,101 @@ package com.example.calculator
 
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.view.View
+import android.widget.Button
+import com.example.calculator.databinding.ActivityMainBinding
+import com.example.calculator.databinding.FragmentTextBinding
+import com.google.android.material.snackbar.Snackbar
+
+class MainActivity : AppCompatActivity(R.layout.activity_main) {
+    private lateinit var binding: ActivityMainBinding
+    private val calculator = CommandProcessor()
+    private val textFragment = TextFragment()
```

Comment:
```suggestion
    private val textFragment 
       get() = TextFragment()
```
лучше так, связано с жизненным циклом, у тебя один инстанс фрагмента который может уничтожиться и потом невозможно будет его снова положить в менеджер, будем такое смотреть позже